api/dataset.go

```diff
@@ -123,14 +123,18 @@ func (api *DatasetAPI) getEditions(w http.ResponseWriter, r *http.Request) {
 	logData := log.Data{"dataset_id": id}
 
 	var state string
+	var auth bool
 	if !api.EnablePrePublishView {
 		state = models.PublishedState
 	} else if r.Header.Get(internalToken) != api.internalToken {
-		logData["authenticated"] = false
+		logData["authenticated"] = auth
```

Comment:
this doesn't need duplicating, just put it after the if/else block

Reply:
So the log data `authenticated` does not make logical sense on a non publishing instance as one can never be authenticated; hence it being added to logData in the second and third if block but not the first.